test.md

```diff
@@ -0,0 +1,3 @@
+<!-- This is a file with the text "This file should be ignored by git". -->
+
+## *This file should be ignored by git*
```

Comment:
- [ ] I guess you tracked this file before you git ignored it. To delete it from the origin (repository), you need to follow the steps in this stack overflow article https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

Reply:
Thank you for that usefull link.

Reply:
> Thank you for that usefull link.

You are welcome @danifromecuador 👋